anet-dictionary.yml

```
@@ -327,7 +327,6 @@ fields:
     mimeTypes:
       - application/pdf
       - image/jpeg
-      - image/jpg
       - image/png
       - text/plain
```

I though we also would add support for webp
```suggestion
      - image/webp
      - text/plain
```